doc/Options/z390_options.md

```diff
@@ -0,0 +1,129 @@
+
+Option | Scope | Default | Description  
+-------|-------|---------|--------------
+@filename	| MALE	| Options in Z390.OPT file if found. |	Retrieve additional options from free form text file with default suffix OPT.  Options can be specified delimited by spaces on as many lines as requires.  All characters on a line following * are ignored as comments. The @file option can be nested.  The default path is the program path.  If a file named Z390.OPT exists in the z390 install directory, these options will be applied first as default options.
+ALIGN | A | YES	| Align DS/DC data fields based on type unless explicit length is specified.  If duplication factor is 0, then NOALIGN is ignored.
+ALLOW | M | NO | Allow extensions to HLASM syntax including: <br />1) No quotes required for SETC variables.<br />2) Duplication factor does not require (..).<br />3) Array declarations with variables in expression accepted<br />4) Substring starting beyond end returns null string<br />5) Quoted strings allowed in SETA and SETB expressions<br />6) Allow &var as null string if not found during substitution<br />7) Allow duplicate local and global variable declarations<br />8) Allow AREAD and PUNCH file records greater than 80 characters. <br />The default of NOALLOW insures HLASM compatibility.
+AMODE24 | LE | NO | Set 390 load module options to start in 24 bit address mode
+AMODE31 | LE | YES | Set 390 load module options to start in 31 bit address mode
+ASCII | AE | NO | Generate ASCII versus EBCDIC DC character constants, compare character strings in ASCII versus EBCDIC in macro processor, generate ASCII versus EBCDIC output for UNPK, ED, and EDMK.    Note ASCII mode is not mainframe compatible and requires careful review of program to insure no EBCDIC immediate compare constants such as X'40', 64, X'F0', 240 etc. are used and that there are no assumptions about EBCDIC versus ASCII collating sequences which have numbers and letters reversed, and EBCDIC letters are non-contiguous.  Typically HLASM compatible programs are run in default EBCDIC mode and DCB RECFM=FT?VT or some other translation option is used to convert between EBCDIC and ASCII where necessary.  Note there are a few instructions affected by ASCII mode: <br />1) ED/EDMK generate ASCII versus EBCDIC but the mask must always be in EBCDIC (masks are usually coded in X'...' format)<br />2)  UNPK generates X'3' versus X'F' in zone field (high order nibble)
+ASM | MA | YES |Run az390 assembler as subtask of mz390 passing BAL.  Note NOASM is for use in pure text processing programs which only use conditional macro code and AREAD/PUNCH with extensions to process ascii text files.  NOASM does not support ordinary symbol attribute tests, OPSYN, or lookahead mode, and requires CHKMAC(0) and CHKSRC(0-2).
+ASSIST | MAE | NO | Enable assembly and execution of 11 new ASSIST extended instructions to simplify I/O for student use.  This option also sets NOLOADHIGH.  See ASSIST Support.
```

Comment:
"ASSIST Support" in the z390.org doc file is a hyperlink to http://www.z390.org/z390_ASSIST_Support.htm.

Reply:
This has been updated to link to the new ASSIST support page in the repo docs. The goal is to not have any docs on the website. All docs will be here (well that's what I was thinking.. I am sure that will cause some discussion)